etl/steps/data/garden/eth/2023-03-15/ethnic_power_relations.py

```diff
@@ -0,0 +1,247 @@
+"""Load the GROWup meadow dataset, the countries_regions dataset and the population dataset to create
+the Ethnic Power Relations garden dataset with regional aggregations."""
+
+import pandas as pd
+from owid.catalog import Dataset, Table
+from structlog import get_logger
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+log = get_logger()
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+# Data processing function for the Ethnic Power-Relations Dataset
+def process_ethnic_power_relations(df: pd.DataFrame) -> pd.DataFrame:
+    # Select only the variables related to the Ethnic Power-Relations Dataset
+    epr_vars = [
+        "egip_groups_count",
+        "excl_groups_count",
+        "regaut_groups_count",
+        "regaut_excl_groups_count",
+        "regaut_egip_groups_count",
+        "rlvt_groups_count",
+        "actv_groups_count",
+        "lpop",
+        "egippop",
+        "legippop",
+        "exclpop",
+        "lexclpop",
+        "discrimpop",
+        "ldiscrimpop",
+        "maxexclpop",
+        "lmaxexclpop",
+        "regautpop",
+        "regautexclpop",
+        "regautegippop",
+        "cntr_relevance",
+        "nstar",
+    ]
+
+    acd2epr_vars = [
+        "onset_ko_eth_flag",
+        "onset_ko_noneth_flag",
+        "onset_ko_terr_eth_flag",
+        "onset_ko_gov_eth_flag",
+        "onset_ko_terr_noneth_flag",
+        "onset_ko_gov_noneth_flag",
+        "incidence_eth_flag",
+        "incidence_noneth_flag",
+        "incidence_terr_eth_flag",
+        "incidence_gov_eth_flag",
+        "incidence_terr_noneth_flag",
+        "incidence_gov_noneth_flag",
+        "onset_do_eth_flag",
+        "onset_do_noneth_flag",
+        "onset_do_terr_eth_flag",
+        "onset_do_gov_eth_flag",
+        "onset_do_terr_noneth_flag",
+        "onset_do_gov_noneth_flag",
+    ]
+
+    df = df[["country", "year"] + epr_vars + acd2epr_vars]
+
+    # Multiply share variables by 100
+    share_vars = [
+        "lpop",
+        "egippop",
+        "legippop",
+        "exclpop",
+        "lexclpop",
+        "discrimpop",
+        "ldiscrimpop",
+        "maxexclpop",
+        "lmaxexclpop",
+        "regautpop",
+        "regautexclpop",
+        "regautegippop",
+    ]
+
+    df[share_vars] *= 100
+
+    return df
+
+
+def load_population() -> pd.DataFrame:
+    # Load population table from key_indicators dataset.
+    ds_indicators = paths.load_dependency("population")
+    tb_population = ds_indicators["population"]
```

Comment:
There's an error with this when running `make test` that I don't know how to solve

Reply:
That's a typing thing, `load_dependency` can return both Dataset and Snapshot so you might tell it what is it that you want (in this case `Dataset`)

This should fix it
```
ds_indicators: Dataset = paths.load_dependency("population")
tb_population = ds_indicators["population"]
```

Reply:
Thank you @Marigold! I will mention this to add the right function in https://github.com/owid/etl/pull/1004